kythe-multi-file-verifier.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -e
```

Comment:
Do we need this script? Can the other one handle both multi and single files?

Reply:
for now, yes but in the coming PR this should be the general case.